### PR TITLE
feat(fluent): add fluent  inputs as individual modules

### PR DESCRIPTION
**Pull request by epetrow**
related to: https://github.com/telerik/kendo-themes/pull/3702

### CODE REVIEW

File: packages/fluent/scss/daterangepicker/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        .k-dateinput {
+            width: $kendo-daterange-picker-input-width;
+        }
+    }
+
+
+    // Angular specific
+    kendo-daterange {
+        // @extend .k-daterangepicker;
+    }
```
Comment (Juveniel) [inline packages/fluent/scss/daterangepicker/_layout.scss:30]: remove this block

**Review by Juveniel — CHANGES_REQUESTED** [review]
Use css vars with fallback in the `_layout`  and `_theme` files and make sure that the values correspond to the correct Fluent design guidelines

File: packages/fluent/scss/datepicker/index.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "datepicker",
+    dependencies: (
+        "input",
+        "popup",
+        "calendar"
+    )
```
Comment (TeyaVes) [inline packages/fluent/scss/datepicker/index.scss:8]: I think we should add a dependency to the Button and Icon here as well

File: packages/fluent/scss/daterangepicker/_layout.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
+@use "../core/" as *;
+@use "../input/_variables.scss" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-daterangepicker--layout() {
+
+    // Daterange picker
+    .k-daterange-picker {
+        width: var( --kendo-daterange-picker-width, #{$kendo-input-width} );
```
Comment (TeyaVes) [inline packages/fluent/scss/daterangepicker/_layout.scss:9]: I think here we should add var which is equal to the kendo-input-width:

var( --kendo-input-width, #{$kendo-input-width} )

In this way we gain two things:
1. the width of the daterangepicker can be changed with the Sass variable $kendo-daterange-picker-width, not only with the CSS var
2. If the --kendo-input-width css var is globally changed, it'll apply here as well

File: packages/fluent/scss/daterangepicker/index.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "daterangepicker",
+    dependencies: (
+        "input",
+        "floating-label",
+        "popup",
+        "icon"
```
Comment (TeyaVes) [inline packages/fluent/scss/daterangepicker/index.scss:8]: add dependency to button here. There is a case where a swap button in rendered between the inputs

File: packages/fluent/scss/dropdowngrid/index.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    dependencies: (
+        "table",
+        "input",
+        "floating-label",
+        "combobox",
+        "popup",
+        "toolbar",
+        "menu",
+        "grid"
+    )
```
Comment (TeyaVes) [inline packages/fluent/scss/dropdowngrid/index.scss:13]: add button dependency

File: packages/fluent/scss/dropdowntree/index.scss [hunk 6]
```diff
@@ -0,0 +1,27 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "dropdowntree",
+    dependencies: (
+        "floating-label",
+        "dropdownlist",
+        "multiselect",
+        "treeview"
+    )
```
Comment (TeyaVes) [inline packages/fluent/scss/dropdowntree/index.scss:9]: add dependency to popup as we are using a var from it

File: packages/fluent/scss/numerictextbox/index.scss [hunk 7]
```diff
@@ -0,0 +1,26 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "numerictextbox",
+    dependencies: (
+        "input",
+        "floating-label",
+        "icon"
+    )
```
Comment (TeyaVes) [inline packages/fluent/scss/numerictextbox/index.scss:8]: add button dependency

File: packages/fluent/scss/datetimepicker/_layout.scss [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+    k-datetimepicker {
+        &[dir="rtl"],
+        .k-rtl & {
+            .k-time-tab .k-datetime-selector {
+                transform: translateX(100%);
+            }
+        }
+    }
+}
```
Comment (TeyaVes) [inline packages/fluent/scss/datetimepicker/_layout.scss:56]: the datetime selector does not look okay in the pictures, I'll take a look at it tmrw

**Review by TeyaVes — CHANGES_REQUESTED** [review]

**Review by Juveniel — APPROVED** [review]

File: packages/fluent/scss/datetimepicker/_layout.scss (hunk 8 above)
Comment (Juveniel) [reply packages/fluent/scss/datetimepicker/_layout.scss:56]: in an ad-hoc discussion we agreed to resolve this with a separate PR for the Calendar

**Review by TeyaVes — APPROVED** [review]

**Pull request merged**